backend/src/main/java/agh/toik/controller/QuestionsApiController.java

```diff
@@ -1,55 +1,71 @@
-package agh.toik.api;
+package agh.toik.controller;
 
+import agh.toik.api.QuestionsApi;
 import agh.toik.model.Question;
 import agh.toik.repo.QuestionRepo;
 import agh.toik.utils.Helper;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import io.swagger.annotations.*;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
 
+import javax.annotation.Generated;
+import javax.validation.constraints.*;
 import javax.validation.Valid;
 import javax.servlet.http.HttpServletRequest;
-import java.io.IOException;
+import java.util.List;
 
-@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2020-05-19T18:19:31.581Z[GMT]")
+@Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2020-05-26T17:58:06.365Z[GMT]")
 @Controller
 public class QuestionsApiController implements QuestionsApi {
 
     private static final Logger log = LoggerFactory.getLogger(QuestionsApiController.class);
-
     private final ObjectMapper objectMapper;
-
     private final HttpServletRequest request;
 
     private QuestionRepo questionRepo;
 
-    @org.springframework.beans.factory.annotation.Autowired
+    @Autowired
     public QuestionsApiController(ObjectMapper objectMapper, HttpServletRequest request, QuestionRepo questionRepo) {
         this.objectMapper = objectMapper;
         this.request = request;
         this.questionRepo = questionRepo;
     }
 
-    public ResponseEntity<Question> addQuestion(@ApiParam(value = "Question object that needs to be added to the db" ,required=true )  @Valid @RequestBody Question entity
-) {
+    public ResponseEntity<Question> addQuestion(@Valid @RequestBody Question entity) {
         entity.setId(Helper.generateID());
         questionRepo.save(entity);
```

Comment:
Missing checking if provided category exists; to match API should be returned 405